roles/openshift_openstack/tasks/provision.yml

```diff
@@ -127,6 +127,21 @@
   when:
   - openshift_openstack_api_lb_provider|default(None) == "haproxy"
 
+# NOTE(ltomasbo): Increasing OpenShift API Loadbalancer timeouts to 500 secs
+# as 50 seconds resulted on failed deployments due to:
+# https://bugzilla.redhat.com/show_bug.cgi?id=1618685
+- name: Octavia OpenShift API listeners timeout correction
+  os_lbaas_listener_timeout:
+    timeout: 500000
```

Comment:
This is only 500 seconds, so 8:20 minutes. I'd make it at least 3600000, so we get just one watcher restart per hour.